pkg/components/cluster-autoscaler/component.go

```diff
@@ -36,7 +36,7 @@ const name = "cluster-autoscaler"
 const chartValuesTmpl = `
 cloudProvider: {{ .Provider }}
 image:
-  tag: v1.17.0
+  tag: v1.18.1
```

Comment:
Actually, we can remove this override completely, as chart now deploys v1.18.1 by default.